html/includes/api_functions.inc.php

```diff
@@ -506,6 +506,31 @@ function list_bgp()
 }
 
 
+function get_bgp()
+{
+    check_is_read();
+
+    $app        = \Slim\Slim::getInstance();
+    $router = $app->router()->getCurrentRoute()->getParams();
+
+    $bgpPeerId        = $router['id'];
```

Comment:
How do you figure out the bgpPeer_id via the API?

What if I want to look up by bgpPeerIdentifier ?

Reply:
This is returned from GET /api/v0/bgp (list_bgp()), and I chose this for get_bgp() as this is the primary key and uniquely identifies the BGP session. I believe bgpPeerIdentifier would be best suited to to an additional filter for list_bgp()? Apologies if I've missed the mark here, this is my first venturing into librenms :)

Reply:
> What if I want to look up by bgpPeerIdentifier ?

That's not a unique reference within the DB.